src/unix/aix-common.c

```diff
@@ -83,12 +83,12 @@ int uv_exepath(char* buffer, size_t* size) {
   struct procsinfo pi;
 
   if (buffer == NULL || size == NULL || *size == 0)
-    return -EINVAL;
+    return UV_ERR(EINVAL);
```

Comment:
Can you change these to `UV_EINVAL`, `UV_ENOMEM`, etc?  Should be a perl one-liner.